pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -192,6 +191,9 @@ func getLatestConfig(url string, iteration int,
 
 	log.Debugf("getLatestConfig(%s, %d)\n", url, iteration)
 
+	// trigger a one sec timer to acquire new certs from cloud
+	triggerFetchCerts(getconfigCtx.zedagentCtx)
```

Comment:
How can triggering fetching the certs once solve it? The certs will roll and you need to be able to re-fetch them. What's the overall approach?




Are the individual microservices responsible to detect that the encryption cert is old? How can they cause a download of the new ones? (Note that for object signining we have an error indicating that the signing cert(s) need to be refreshed, but that is easier because it is trigger in zedagent itself.)

I don't have an issue with initially fetching them (although that could fail if zedcloud is down when zedagent starts). But need to understand the requirements and ensure we know how to address it all. Then the fixes can come in small pieces.